scripts/launch/node.sh

```diff
@@ -119,6 +129,12 @@ if [[ "$node" == "educator" ]]; then
     else
         dscp-educator $common_params $witness_params $educator_params
     fi
+elif [[ "$node" == "multi-educator" ]]; then
+    if [ -z "$(which dscp-multi-educator)" ]; then
+        stack exec "dscp-multi-educator" -- $common_params $witness_params $educator_params
+    else
+        dscp-multi-educator $common_params $witness_params $educator_params
```

Comment:
But this doesn't work, one should pass `$multi_educator_params` instead of `$educator_params` here :thinking: